Description: disable locale specific tests
 On buildd network locales might not be available.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2022-03-31

---


--- thrift-0.16.0.orig/lib/cpp/test/ToStringTest.cpp
+++ thrift-0.16.0/lib/cpp/test/ToStringTest.cpp
@@ -44,6 +44,7 @@ BOOST_AUTO_TEST_CASE(base_types_to_strin
 // NOTE: Currently (as of 2021.08.12) the locale-based tests do not work on
 // Windows in the AppVeyor Thrift CI build correctly. Therefore disabled on
 // Windows:
+/*
 #ifndef _WIN32
 BOOST_AUTO_TEST_CASE(locale_en_US_int_to_string) {
 #ifdef _WIN32
@@ -53,7 +54,9 @@ BOOST_AUTO_TEST_CASE(locale_en_US_int_to
 #endif
   BOOST_CHECK_EQUAL(to_string(1000000), "1000000");
 }
+*/
 
+/*
 BOOST_AUTO_TEST_CASE(locale_de_DE_floating_point_to_string) {
 #ifdef _WIN32
   std::locale::global(std::locale("de-DE.UTF-8"));
@@ -65,6 +68,7 @@ BOOST_AUTO_TEST_CASE(locale_de_DE_floati
   BOOST_CHECK_EQUAL(to_string(1.5L), "1.5");
 }
 #endif
+*/
 
 BOOST_AUTO_TEST_CASE(empty_vector_to_string) {
   std::vector<int> l;
